src/src/PluginStructs/P036_data_struct.cpp

```diff
@@ -783,40 +1050,99 @@ uint8_t P036_data_struct::display_scroll(ePageScrollSpeed lscrollspeed, int lTas
         log += F(" CharsRemoved: "); log += iCharToRemove;
         addLog(LOG_LEVEL_INFO, log);
         log.clear();
-        log += F(" -> Changed to: "); log += ScrollingPages.LineIn[j];
-        log += F(" Length: "); log += ScrollingPages.LineIn[j].length();
-        log += F(" PixLength: "); log += display->getStringWidth(ScrollingPages.LineIn[j]);
+        log += F(" -> Changed to: "); log += ScrollingPages.In[j].SPLcontent;
+        log += F(" Length: "); log += ScrollingPages.In[j].SPLcontent.length();
+        display->setFont(FontSizes[LineSettings[ScrollingPages.In[j].SPLidx].fontIdx].fontData);
```

Comment:
I think this is an error.
You perform actions which depend on the set log level.

Reply:
It's not an error.
display->setFont() must be used to calculate the display->getStringWidth() according to the individual font.
The StringWidth should just be logged, nothing else.
Later the setting of display->setFont() is overwritten by the correct values for this line.